<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PC-liangxuqing" name="com.xyuan.ngtest.ps.comp.adminsanction" tests="1" failures="0" timestamp="10 Oct 2017 07:12:59 GMT" time="0.143" errors="1">
  <testcase name="test" time="0.143" classname="com.xyuan.ngtest.ps.comp.adminsanction">
    <error type="java.lang.AssertionError" message="
Expected: a string containing &quot;\&quot;trackValueId\&quot;:130728&quot;
     but: was &quot;{&quot;sta&quot;:1,&quot;data&quot;:{&quot;result&quot;:[{&quot;decisionTime&quot;:&quot;2017-10-09 17:36:24&quot;,&quot;decisionOrgan&quot;:&quot;就是这样&quot;,&quot;docNumber&quot;:&quot;1&quot;,&quot;type&quot;:&quot;11&quot;,&quot;punishContent&quot;:&quot;有履行能力而拒不履行生效法律文书确定义务&quot;}],&quot;totalCount&quot;:1,&quot;pageNo&quot;:1,&quot;pageSize&quot;:20,&quot;totalPage&quot;:1},&quot;msg&quot;:&quot;ok&quot;}&quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "\"trackValueId\":130728"
     but: was "{"sta":1,"data":{"result":[{"decisionTime":"2017-10-09 17:36:24","decisionOrgan":"就是这样","docNumber":"1","type":"11","punishContent":"有履行能力而拒不履行生效法律文书确定义务"}],"totalCount":1,"pageNo":1,"pageSize":20,"totalPage":1},"msg":"ok"}"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.xyuan.ngtest.MyAssert.MyAssert.asserting(MyAssert.java:12)
	at com.xyuan.ngtest.ps.comp.adminsanction.test(adminsanction.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:369)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- test -->
</testsuite> <!-- com.xyuan.ngtest.ps.comp.adminsanction -->
